src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,139 @@
+html,
```

Comment:
I think it does make sense to use any package instead of copying the reset to every your new project.

Reply:
You can also try to practice creating your own package if you wish to leave this content as is

Reply:
Done. I have created new package  scss-reset-by-igor-saroko for reset.scss file.